python/tank/bootstrap/manager.py

```diff
@@ -29,6 +29,12 @@ def __init__(self, sg_user):
         :param sg_user: Authenticated Shotgun User object
         :type sg_user: :class:`~sgtk.authentication.ShotgunUser`
         """
+        # Begin writing log to disk.
+        # Because this is a pre-launch state and we don't
+        # know yet what environment we are launching into,
+        # call the log 'bootstrap'.
+        LogManager().initialize_base_file_logger("bootstrap")
```

Comment:
this is now gone following our discussion - instead the bootstrap code will init. I have updated the tank command to follow this pattern.